specification/cognitiveservices/OpenAI.Inference/models/images/images.tsp

```diff
@@ -10,80 +10,170 @@ using TypeSpec.Versioning;
 
 namespace Azure.OpenAI;
 
-@doc("The desired size of the generated images. Must be one of 256x256, 512x512, or 1024x1024.")
+@doc("The desired size of generated images.")
 @added(ServiceApiVersions.v2023_06_01_Preview)
 enum ImageSize {
-  @doc("Image size of 256x256.")
-  Size256x256: "256x256",
+  @doc("""
+  Very small image size of 256x256 pixels.
+  Only supported with dall-e-2 models.
+  """)
+  size256x256: "256x256",
+
+  @doc("""
+  A smaller image size of 512x512 pixels.
+  Only supported with dall-e-2 models.
+  """)
+  size512x512: "512x512",
+
+  @doc("""
+  A standard, square image size of 1024x1024 pixels.
+  Supported by both dall-e-2 and dall-e-3 models.
+  """)
+  @added(ServiceApiVersions.v2023_12_01_Preview)
+  size1024x1024: "1024x1024",
 
-  @doc("Image size of 512x512.")
-  Size512x512: "512x512",
+  @doc("""
+  A wider image size of 1024x1792 pixels.
+  Only supported with dall-e-3 models.
+  """)
+  @added(ServiceApiVersions.v2023_12_01_Preview)
+  size1792x1024: "1792x1024",
 
-  @doc("Image size of 1024x1024.")
-  Size1024x1024: "1024x1024",
+  @doc("""
+  A taller image size of 1792x1024 pixels.
+  Only supported with dall-e-3 models.
+  """)
+  @added(ServiceApiVersions.v2023_12_01_Preview)
+  size1024x1792: "1024x1792",
+}
+
+@doc("""
+An image generation configuration that specifies how the model should prioritize quality, cost, and speed.
+Only configurable with dall-e-3 models.
+""")
+@added(ServiceApiVersions.v2023_12_01_Preview)
+enum ImageGenerationQuality {
+  @doc("""
+  Requests image generation with standard, balanced characteristics of quality, cost, and speed.
+  """)
+  standard: "standard",
+
+  @doc("""
+  Requests image generation with higher quality, higher cost and lower speed relative to standard.
+  """)
+  hd: "hd",
+}
+
+@doc("""
+An image generation configuration that specifies how the model should incorporate realism and other visual characteristics.
+Only configurable with dall-e-3 models.
+""")
+@added(ServiceApiVersions.v2023_12_01_Preview)
+enum ImageGenerationStyle {
+  @doc("""
+  Requests image generation in a natural style with less preference for dramatic and hyper-realistic characteristics.
+  """)
+  natural: "natural",
+
+  @doc("""
+  Requests image generation in a vivid style with a higher preference for dramatic and hyper-realistic
+  characteristics.
+  """)
+  vivid: "vivid",
 }
 
 @doc("The format in which the generated images are returned.")
 @added(ServiceApiVersions.v2023_06_01_Preview)
 enum ImageGenerationResponseFormat {
   @doc("Image generation response items should provide a URL from which the image may be retrieved.")
-  Url: "url",
+  url: "url",
 
   @doc("Image generation response items should provide image data as a base64-encoded string.")
-  Base64: "b64_json",
+  base64: "b64_json",
```

Comment:
I prefer this the way it is, but should we really drop `_json`? This rings a similar bell as keeping `n` as "n" since people are already used to it.